ui/src/core/xibo-cms.js

```diff
@@ -479,8 +479,8 @@ function XiboInitialise(scope) {
         $input.hide();
 
         const commandTypes = {
-            'freetext': 'freeText',
```

Comment:
I think we need a translation for this one (probably not the other one as its a product name).